scripts/post_build.js

```diff
@@ -50,7 +50,23 @@ function movePackageJSON() {
   );
 }
 
+function copyCJSPackageJSON() {
+  const packageJSONDistPath = path.resolve(getDistPath('/cjs'), 'package.json');
+  fs.writeFileSync(
+    packageJSONDistPath,
+    JSON.stringify({
+      ...packageTemplate,
+      main: 'index.js',
+      type: 'commonjs',
```

Comment:
@sravan-s Is there anything we want to add or omit here?

Reply:
Maybe (?) Even tho they shouldnt do anything ~
    peerDependencies,
    dependencies,
    devDependencies,
    browserslist,

Reply:
https://github.com/sendbird/sendbird-uikit-react/pull/687/commits/61ae6c3684445ba1e04559cac06c20e7e97560ce How about this? 
```
{
  "version": "3.6.3",
  "name": "@sendbird/uikit-react",
  "description": "Sendbird UIKit for React: A feature-rich and customizable chat UI kit with messaging, channel management, and user authentication.",
  "keywords": [
    "sendbird",
    "uikit",
    "react",
    "chat",
    "messaging",
    "real-time",
    "UI components",
    "user authentication",
    "channel management",
    "SDK integration",
    "customizable",
    "feature-rich",
    "social app",
    "customer support",
    "API"
  ],
  "type": "commonjs",
  "main": "index.js",
  "style": "dist/index.css",
  "typings": "index.d.ts",
  "module": "index.js",
  "repository": {
    "type": "git",
    "url": "git+git@github.com:sendbird/sendbird-uikit-react-sources.git"
  },
  "author": "SendBird <support@sendbird.com>",
  "license": "SEE LICENSE IN LICENSE.md",
  "bugs": {
    "url": "https://community.sendbird.com"
  },
  "homepage": "https://sendbird.com",
  "peerDependencies": {
    "react": "^16.8.6 || ^17.0.0 || ^18.0.0",
    "react-dom": "^16.8.6 || ^17.0.0 || ^18.0.0"
  },
  "dependencies": {
    "@sendbird/chat": "^4.9.2",
    "@sendbird/uikit-tools": "0.0.1-alpha.40",
    "css-vars-ponyfill": "^2.3.2",
    "date-fns": "^2.16.1",
    "dompurify": "^3.0.1",
    "prop-types": "^15.7.2"
  },
  "browserslist": [
    "defaults",
    "IE 11"
  ]
}
```

Reply:
AFAIK, we just need one line of package.json `{ type: "commonjs" }`
https://nodejs.org/dist/latest-v18.x/docs/api/packages.html#determining-module-system
<img width="1433" alt="image" src="https://github.com/sendbird/sendbird-uikit-react/assets/26326015/3736282d-6a92-4900-9367-1bab9ccaa530">